activestate.macos.yaml

```diff
@@ -0,0 +1,70 @@
+constants:
```

Comment:
This file has a lot of stuff in it for very little change compared to the runners they're overriding. Please try to dial this back to the point that it only contains essentials.